net/scripts/gce-provider.sh

```diff
@@ -171,18 +173,23 @@ cloud_CreateInstances() {
   declare optionalAdditionalDiskSize="${11}"
   declare optionalPreemptible="${12}"
   #declare sshPrivateKey="${13}"  # unused
+  declare optionalImageName="${14}"
 
-  if $enableGpu; then
-    # Custom Ubuntu 20.04 LTS image with CUDA 10.2 installed
-    #
-    # Unfortunately this image is not public.  When this becomes an issue, use
-    # the stock Ubuntu 20.04 image and programmatically install CUDA after the
-    # instance boots
-    #
-    imageName="ubuntu-2004-focal-v20201211-with-cuda-10-2 --image-project principal-lane-200702"
+  if [[ -n $optionalImageName ]]; then
+    imageName=$optionalImageName
   else
-    # Upstream Ubuntu 20.04 LTS image
-    imageName="ubuntu-2004-focal-v20201201 --image-project ubuntu-os-cloud"
+    if $enableGpu; then
+      # Custom Ubuntu 20.04 LTS image with CUDA 10.2 installed
+      #
+      # Unfortunately this image is not public.  When this becomes an issue, use
+      # the stock Ubuntu 20.04 image and programmatically install CUDA after the
+      # instance boots
+      #
+      imageName="ubuntu-2004-focal-v20201211-with-cuda-10-2 --image-project principal-lane-200702"
+    else
+      # Upstream Ubuntu 20.04 LTS image
+      imageName="ubuntu-2004-focal-v20201201 --image-project ubuntu-os-cloud"
```

Comment:
why not upgrade this to `ubuntu-2004-focal-v20220419`?

Reply:
I would prefer to update it as well, [I've asked this question on discord back in April](https://discord.com/channels/428295358100013066/439194979856809985/966721835586699296). Seems like only @t-nelson  was interested why do I want to do that.
Anyway, I'm happy to remote image option and just upgrade image if we agree here to do so